src/inc/quic_platform_winuser.h

```diff
@@ -736,7 +740,9 @@ CxPlatEventQInitialize(
     _Out_ CXPLAT_EVENTQ* queue
     )
 {
-    return (*queue = CreateIoCompletionPort(INVALID_HANDLE_VALUE, NULL, 0, 1)) != NULL;
+    queue->CompletionKey = 0;
+
+    return (queue->IoCompletionPort = CreateIoCompletionPort(INVALID_HANDLE_VALUE, NULL, 0, 1)) != NULL;
```

Comment:
nit: As this line becomes more complex, please split the assignment and the comparison/return.